celesta-sql/src/main/java/ru/curs/celesta/score/discovery/ScoreByScorePathDiscovery.java

```diff
@@ -49,15 +50,29 @@ public ScoreByScorePathDiscovery(String scorePath) {
 
     private Set<Resource> discoverScore(final File scoreDir) {
         try {
-            return Files.walk(scoreDir.toPath(), FileVisitOption.FOLLOW_LINKS)
+            Path scorePath = scoreDir.toPath().toAbsolutePath();
+            return Files.walk(scorePath, FileVisitOption.FOLLOW_LINKS)
                     .filter(path -> Files.isRegularFile(path) && path.toString().endsWith(".sql"))
-                    .map(Path::toAbsolutePath)
-                    .map(Path::toFile)
-                    .map(FileResource::new)
+                    .map(p -> {
+                        return new FileResource(p.toFile(), getNamespaceFromPath(scorePath.relativize(p)));
```

Comment:
It will be a bit more clear if you use expression lambda
```java
p -> new FileResource(...)
```
without braces & `return`

Reply:
Done.